be/src/vec/common/hash_table/string_hash_table.h

```diff
@@ -373,8 +377,6 @@ class StringHashTable : private boost::noncopyable {
                 }
                 }
             }
-            while (need_switch_to_next)
```

Comment:
what the code mean ？ rethink to delete code is safe？

Reply:
double checked it. should write in LOG(FATAL, ...). I'll move it to another pr.